src/add-to-project.ts

```diff
@@ -39,6 +59,13 @@ interface ProjectV2AddDraftIssueResponse {
 export async function addToProject(): Promise<void> {
   const projectUrl = core.getInput('project-url', {required: true})
   const ghToken = core.getInput('github-token', {required: true})
+  const fields = core.getInput('fields', {required: true})
+  const fieldsObj = fields.split('\n').reduce((acc, f) => {
```

Comment:
Fields will be already split if we define them as a multiline input[^1]

[^1]: https://github.com/actions/toolkit/blob/1589a5c066fbe34ed1c828e9b81b9f206f9a3a40/packages/core/src/core.ts#L140-L161